roles/setup-local/templates/acsf.aliases.drushrc.php

```diff
@@ -2,8 +2,8 @@
 ### Wildcard for ACSF: Stanford & Cardinal
 ###
 ### Examples:
-### @acsf.[stack].[inventory_hostname]
-### @acsf.[dev|test].[stack].[inventory_hostname]
+### @acsf.[stack].[sitename]
```

Comment:
Shouldn't this stay as `inventory_hostname` here, since we're renaming that variable?

Reply:
Yes!  I updated this file, but forgot to change the variable name back to `inventory_hostname`.